task_manager/tests/test_task.py

```diff
@@ -0,0 +1,190 @@
+import pytest
+from django.template.response import TemplateResponse
+from django.test import Client
+from task_manager.statuses.models import Status
+from task_manager.tasks.forms import TaskForm
+from task_manager.tasks.models import Task
+from task_manager.users.models import ServiceUser
+
+pytestmark = pytest.mark.django_db
+
+
+def test_tasks_list(client: Client, service_user: ServiceUser, task: Task) -> None:
+    # given
+    client.force_login(service_user)
+    tasks = Task.objects.all()
+    route = "/tasks/"
+
+    # when
+    response = client.get(route)
+    csrf_token = response.context.get("csrf_token")
+    context = {"csrf_token": csrf_token, "tasks": tasks}
+
+    # then
+    expected_html = TemplateResponse(
+        response.wsgi_request, "tasks_list.html", context=context
+    ).render()
+    assert expected_html.content.decode() == response.content.decode()
+    assert response.status_code == 200
+
+
+def test_create_task_get(client: Client, service_user: ServiceUser) -> None:
+    # given
+    client.force_login(service_user)
+    route = "/tasks/create/"
+
+    # when
+    response = client.get(route)
+    csrf_token = response.context.get("csrf_token")
+    form = TaskForm()
+    context = {"form": form, "csrf_token": csrf_token}
+
+    # then
+    expected_html = TemplateResponse(
+        response.wsgi_request, "create_task.html", context=context
+    ).render()
+    assert expected_html.content == response.content
+    assert response.status_code == 200
+
+
+def test_create_task_post(
+    client: Client, service_user: ServiceUser, status: Status
+) -> None:
+    # given
+    client.force_login(service_user)
+    route = "/tasks/create/"
+    form_data = {
+        "name": "fix",
+        "description": "",
+        "status": status.id,
+        "executor": service_user.id,
+    }
+
+    # when
+    response = client.post(route, data=form_data)
+
+    # then
+    assert Task.objects.get(id=1).name == "fix"
```

Comment:
wdyt, does it make sense to compare more fields? does this test do enough comparisons so we know exactly _what object_ was created?